livingdoc-converters/src/test/kotlin/utils/Extensions.kt

```diff
@@ -0,0 +1,7 @@
+package utils
+
+import org.livingdoc.api.conversion.TypeConverter
+
+fun  <T> TypeConverter<T>.convert(value: String): T? {
+        return convert(value, null, null)
+}
```

Comment:
I'm not sure if this is the best way to go about it. Its a little bit more work to add the `, null, null` to each affected part of the code, but I needed way to long to remember why this even existed and why it adds some strange diffs to the converter tests (e.g. import on package only)

Reply:
Easier but less readable. 
Changing it in favor of test as documentation